lib/systems/all.nix

```diff
@@ -0,0 +1,336 @@
+# These can be passed to nixpkgs as either the `localSystem` or
```

Comment:
You need to add this to `lib/systems/default.nix` ([here](https://github.com/nixos/nixpkgs/blob/d0c5f0c0b8063901b1067c7333ddf91280816cdf/lib/systems/default.nix#L9)). It doesn't seem to do any magic

Reply:
> Why plural _"PRs"_? Better to do it in one swoop to avoid more new references being added.

Okay!

> Did you mean `trace` instead of `throw`?

Ah, I was wondering how to have nixpkgs print a "warning".  I assumed `trace` was supposed to be for debugging only.  If it is okay to commit expressions that use it to print warnings, then I would like to add that as an *additional* step:

At some point in the future I would like to gradually migrate towards removing the alias, in steps:

1. merge this commit
2. replace the examples.nix alias with a trace
3. replace the trace with a throw
4. drop examples.nix entirely

...with no more than one step taken in each twice-per-year release.

Reply:
[done](https://github.com/NixOS/nixpkgs/pull/175341/commits/a3a8eb0b7666f343e32db38123f9f8ac022c2016).

Reply:
I meant:
```nix
  all = import ./all.nix { inherit lib; };
  examples = import ./examples.nix { inherit lib; };
```
We can't change the file structure yet since people might be importing `lib/systems/examples.nix` directly for whatever reason, so this file should just have a 1-1 mapping of the directory structure.